jeeves/jobs.py

```diff
@@ -4,6 +4,16 @@
 import datetime
 
 
+def _get_stage_failure(build_stages):
```

Comment:
Function looks good but why do you have a "_" at the front of the name?

Reply:
just as a convention to declare private function

Reply:
Let's remove it